### DISCUSSION TITLE
Issue with sending to dymo printer

### DISCUSSION THREAD

**Discussion by bertrandyvernault** [apenab/react-dymo#36] [General]
Hello,

Thank you a lot for your work on this lib !

I'm actually facing an issue on some computer, when i'm sending my xml to print, i'm receiving an error and on the devTools, the response we have is :

a key is missing in the dictionary (or something like this).

Sometimes remove dymo softawere and reinstalling works but sometimes not...

I'm wondering if you have already facing this issue ?

**Comment by apenab** [comment]
Hi, thank you. I'm not facing this issue, but if you send me more details(OS, Stack error, ect..) i can try help you

**Comment by bertrandyvernault** [comment]
This issue happened on Windows. I'm using on my windows and it's working find.

I have two leads on the problem :
* A issue when installing `dymo connect` when users already had  the software `dymo label` installed
* The version of the dymo software, seems to work on 1.3 but not 1.4.

I found this : https://www.dymo.com/compatibility-chart.html maybe, it's juste some incompatibility...

I don't have a Stack error, on a dev tools from the user browser, we have the message :
`"[0] The given key was missing from the dictionary."`
